### ISSUE TITLE
Implementation Details

### ISSUE THREAD

**Issue by qiuxiaoyu9954** [qikunxun/PCT#1]
Hi! Thanks for the open source code! I've been reproducing your experiment recently. I would like to ask, is the experimental setting in the paper only full-data? What is the training method in the few-shot scenario, Including hyperparameters such as batch size and learning rate?

**Comment by qikunxun** [comment]
> Hi! Thanks for the open source code! I've been reproducing your experiment recently. I would like to ask, is the experimental setting in the paper only full-data? What is the training method in the few-shot scenario, Including hyperparameters such as batch size and learning rate?

You can find the code for the few-shot scenario in train_xnli_fewshot.py, all the hyper-parameters used in our paper are provided in this file.

**Comment by qiuxiaoyu9954** [comment]
Okay, thank you! I have a question is whether the warmup mechanism used is linear? Can I replace it with transformers.get_linear_schedule_with_warmup? (Because I am using the pytorch framework.）

**Comment by qikunxun** [comment]
> Okay, thank you! I have a question is whether the warmup mechanism used is linear? Can I replace it with transformers.get_linear_schedule_with_warmup? (Because I am using the pytorch framework.）

Yes, the warmup mechanism used in our code is linear. I think you can replace it with transformers.get_linear_schedule_with_warmup.

**Comment by qiuxiaoyu9954** [comment]
OK! Thank you very much!
